uptime_service_validation/coordinator/server.py

```diff
@@ -1,74 +1,257 @@
 import logging
 import sys
-from kubernetes import client
+from kubernetes import client, config
 import os
-from datetime import datetime
+from datetime import datetime, timezone
 import subprocess
 import time
-
-# Set environment variables
-# Get the values of your-image and tag from environment variables
-worker_image = os.environ.get("WORKER_IMAGE", "busybox")
-worker_tag = os.environ.get("WORKER_TAG", "latest")
+import string
+import random
 
 
 # Format datetime such as it is accepted by the stateless validator
 def datetime_formatter(dt):
     return dt.strftime("%Y-%m-%d %H:%M:%S.%f")[:-5] + "+0000"
 
 
-def setUpValidatorPods(time_intervals, jobs, logging, worker_image, worker_tag):
-    # Create a Kubernetes API client
-    api = client.BatchV1Api()
+def setUpValidatorPods(time_intervals, logging, worker_image, worker_tag):
+    # Configuring Kubernetes client
+    config.load_incluster_config()
+
+    api_core = client.CoreV1Api()
+    api_batch = client.BatchV1Api()
+
+    namespace = (
+        open("/var/run/secrets/kubernetes.io/serviceaccount/namespace").read().strip()
+    )
+
+    platform = os.environ.get("PLATFORM")
+    testnet = os.environ.get("TESTNET")
+
+    service_account_name = f"{platform}-{testnet}-delegation-verify"
+
+    worker_cpu_request = os.environ.get("WORKER_CPU_REQUEST")
+    worker_memory_request = os.environ.get("WORKER_MEMORY_REQUEST")
+    worker_cpu_limit = os.environ.get("WORKER_CPU_LIMIT")
+    worker_memory_limit = os.environ.get("WORKER_MEMORY_LIMIT")
+
+    # List to keep track of job names
+    jobs = []
+
+    # List to keep track of config maps
+    configmaps = []
+
     for index, mini_batch in enumerate(time_intervals):
-        job_name = f"zk-validator-{datetime.now().strftime('%y-%m-%d-%H-%M')}-{index}"  # ZKValidator
-        jobs.append(job_name)
+        # Define the environment variables
+        env_vars = [
+            client.V1EnvVar(name="CQLSH", value=os.environ.get("CQLSH")),
+            client.V1EnvVar(
+                name="AWS_ROLE_SESSION_NAME",
+                value=os.environ.get("AWS_ROLE_SESSION_NAME").rstrip("-coordinator"),
+            ),
+            client.V1EnvVar(
+                name="CASSANDRA_HOST", value=os.environ.get("CASSANDRA_HOST")
```

Comment:
I wonder if the whole script logic that is doing nslookup below couldn't be just replaced by something like this here: 
```suggestion
                name="CASSANDRA_HOST", value=socket.gethostbyname(os.environ.get("CASSANDRA_HOST"))
```
`socket.gethostbyname` - returns IP of the host, and that would be just passed as `CASSANDRA_HOST`.
I'm doing something like this in e2e test and empoyed it in https://github.com/MinaFoundation/uptime-service-validation/pull/28 for `setUpValidatorProcesses` method that is used in e2e test.